admin-dev/themes/new-theme/js/app/utils/reset_search.js

```diff
@@ -29,8 +29,10 @@
 
 const $ = global.$;
 
-const init = function resetSearch(url) {
+const init = function resetSearch(url, redirectUrl) {
     $.post(url);
+    console.log(redirectUrl);
```

Comment:
tsssss

Reply:
wasn't me, I saw you appending my commit OMG :trollface:

Reply:
It's @tomlev fault? :D